packages/terra-date-picker/src/DateUtil.jsx

```diff
@@ -142,6 +143,98 @@ class DateUtil {
     return undefined;
   }
 
+  /**
+   * Returns an object consisting of placeholder strings and a delimiter,
+   * based on the date format.
+   * @param {string} variant - Variant string obtained from getDateFormat.
+   * @param {string} placeholder - Passed on placeholder prop string.
+   */
+  static getPlaceholderValues(variant, placeholder) {
```

Comment:
My thoughts - This seem like it could be easily pass in the date separator that we load from translations and we can use it to split the placeholder with that value:

```
// call like
getPlaceholderValues(dateOrder, placeholder, separator)

// use like
static getPlaceholderValues(dateOrder, placeholder, intl.formatMessage({ id: 'Terra.datePicker.seperator' })) {
     const placeholderParts = placeholder.split(separator);

      let day;
      let mouth;
      let year;

      switch(dateOrder) {
          case 'DMY':
               [day, month, year] = placeholderParts;
                break;
          case 'MDY':
               [month, day, year] = placeholderParts;
                break;
          case 'YMD':
               [year, month, day] = placeholderParts;
                break;
         default':
               [year, day, month] = placeholderParts;
      }
      return { year, day, month }
}

```